sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -476,7 +476,7 @@ def __init__(self, label='ToList'):
     super(ToList, self).__init__(label)
 
   def expand(self, pcoll):
-    return pcoll | core.CombineGlobally(self.label, ToListCombineFn())
+    return pcoll | self.label >> core.CombineGlobally(ToListCombineFn())
```

Comment:
To answer that question I think label1 will override as that is applied after the label2 computation is already done.